src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageCreateActionImpl.java

```diff
@@ -179,4 +183,12 @@ public MessageCreateAction deadline(long timestamp)
     {
         return (MessageCreateAction) super.deadline(timestamp);
     }
+
+    @NotNull
+    @Override
+    public MessageCreateAction setSilent(boolean silent)
```

Comment:
This is handled by the mixin implementation already

Reply:
Done.